tests/test-http-api/test-config.js

```diff
@@ -143,6 +148,69 @@ describe('config', () => {
         done()
       })
     })
+
+    describe('/config/replace', () => {
+      it('returns 400 if no config is provided', (done) => {
+        const form = new FormData()
+        const headers = form.getHeaders()
+
+        streamToPromise(form).then(payload => {
```

Comment:
are you using Standard 6? It forces you to put parentheses on single arg as well